src/IIIF.php

```diff
@@ -344,24 +344,74 @@ public function buildCanvasWithPages ($index, $uri, $canvasData) {
 
     }
 
+    private function buildTranscript($language_code, $page, $target) {
+        $datastream = $this->url . '/collections/islandora/object/' . $this->pid . '/datastream/';
+        if ($language_code == "es") :
+            $transcript_datastream = "TRANSCRIPT-ES";
+            $transcript_label = "Captions in Spanish";
```

Comment:
This might be overkill, but thinking ahead to usability for Spanish language folks.
```suggestion
            $transcript_label = "Subtítulos en español";
```

Reply:
sounds good